src/services/apiClient.ts

```
@@ -0,0 +1,55 @@
+  ): Promise<Response>;
+
+  <Response>(
+    options: Omit<ApiClientProps<Response>, 'method'> & { method: 'GET' },
+  ): Promise<Response>;
+}
+
+const apiClient: ApiClientType = async <Response, RequestBody>({
+  method,
+  URI,
+  body,
+}: ApiClientProps<RequestBody>): Promise<Response | void> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)